src/main/java/com/epam/java/pdp/figures/Shape.java

```diff
@@ -18,44 +18,53 @@ public Shape(Point... points) {
 
     boolean isTriangle() {
         if (points.size() == 3) {
-            System.out.println("This is triangle");
+//            calculateLineLengths();
+            arePointsOnTheSingleLine();
+            System.out.println("This is a triangle");
```

Comment:
Shape is a triange every time it contains 3 points. Is that correct?

Reply:
added validation if points are on the same line

Reply:
That will NOT work if method isTriangle will be the only one with public access. 
Task: mark all rest methods as a private ones.